### PR TITLE
Small ONI base fixes

### PR DESCRIPTION
**Pull request by erty80s**
-The fences outside the base no longer have weird collision issues and should work as intended

-Fixed the issue where the Yggdrasil mech parts didn't stick to the chasis and the torso spawned invisible.

### CODE REVIEW

**Review by BDpuffy420 — DISMISSED** [review]

File: maps/Exoplanet Research/ExoResearch_1.dmm [hunk 1]
```diff
@@ -145,7 +145,7 @@
...
 "cP" = (/obj/structure/window/reinforced{dir = 4},/obj/machinery/portable_atmospherics/hydroponics{closed_system = 1; name = "isolation tray"},/turf/simulated/floor/tiled/dark,/area/exo_research_facility{base_turf = /turf/simulated/floor/exoplanet/desert; icon_state = "apmaint"; name = "Research Facility Labs"; requires_power = 0; tag = "icon-apmaint"})
 "cQ" = (/obj/machinery/portable_atmospherics/hydroponics,/turf/simulated/floor/tiled/dark,/area/exo_research_facility{base_turf = /turf/simulated/floor/exoplanet/desert; icon_state = "apmaint"; name = "Research Facility Labs"; requires_power = 0; tag = "icon-apmaint"})
-"cR" = (/obj/structure/electric_fence/ns{tag = "icon-fence-ns2"; icon_state = "fence-ns2"},/turf/simulated/floor/pavement/empty,/area/exo_research_facility/outer)
+"cR" = (/obj/structure/electric_fence/ns{bound_height = 96; step_x = 0; step_y = 0},/turf/simulated/floor/exoplanet/desert,/area/exo_research_facility/outer)
```
Comment (bloxgate) [inline maps/Exoplanet Research/ExoResearch_1.dmm]: Both step_x and step_y need to be removed here.

File: maps/Exoplanet Research/ExoResearch_1.dmm [hunk 2]
```diff
@@ -285,208 +285,211 @@
...
-"js" = (/obj/machinery/telecomms/relay/long_range_planetary,/turf/simulated/floor/exoplanet/desert,/area/exoplanet/desert{dynamic_lighting = 0; luminosity = 1; requires_power = 0})
-"jt" = (/turf/unsimulated/floor/rock2,/area/exoplanet/desert{dynamic_lighting = 1; icon_state = "ai_server"; luminosity = 0; name = "\improper Planetary surfacecave"; requires_power = 0; tag = "icon-ai_server"})
-"ju" = (/obj/effect/floor_decal/asteroid,/turf/unsimulated/floor/rock2{density = 1; opacity = 1},/area/exoplanet/desert{dynamic_lighting = 1; icon_state = "ai_server"; luminosity = 0; name = "\improper Planetary surfacecave"; requires_power = 0; tag = "icon-ai_server"})
+"fB" = (/obj/structure/electric_fence/ns{bound_height = 96; step_x = 0; step_y = 0},/turf/simulated/floor/exoplanet/desert,/area/exo_research_facility{base_turf = /turf/simulated/floor/exoplanet/desert; icon_state = "apmaint"; name = "Research Facility Interior up"; requires_power = 0; tag = "icon-apmaint"})
```
Comment (bloxgate) [inline maps/Exoplanet Research/ExoResearch_1.dmm]: And again

**Review by bloxgate — CHANGES_REQUESTED** [review]
Map contains step_x and step_y, movement code will break if merged.

**Comment by Flaksim** [comment]
Hold on, why is exoresearch_1 deleted? Shit like that can break our entire load order with the flaky things we have in to link maps.

**Comment by erty80s** [comment]
hmm it still seems to fail the travis check, anyone know why?

**Comment by BDpuffy420** [comment]
It passed , it’s just Blox’s review

**Review by bloxgate — APPROVED** [review]

**Comment by bloxgate** [comment]
Whoops, sorry about the delay. LGTM.

That said, have @Flaksim's questions been resolved?

**Comment by erty80s** [comment]
Yes, basically what happened was that I made a copy of exoresearch and placed it in the wrong folder. Anyways, everything should be in order and ready to go.

**Pull request merged**